pages/developers/style.md

```diff
@@ -517,5 +517,40 @@ If you have shell scripts, you can protect against common mistakes using [shellc
   - id: shellcheck
 ```
 
+## PyLint (noisy)
+
+PyLint is very opinionated, with a high signal-to-noise ratio. However, by
+limiting the default checks or by starting off a new project using them, you
+can get some very nice linting, including catching some problematic code that
+otherwise is hard to catch. PyLint is generally not a good candidate for
+pre-commit, since it needs to have your package installed - it is less static
+of a static check than flake8. Here is a suggested pyproject.toml entry to get
```

Comment:
By "less static of a static check than flake8" you mean "less a static check than Flake8"? Note that I believe we have been writing Flake8 with a capital F.

Reply:
```suggestion
of check than Flake8. Here is a suggested pyproject.toml entry to get
```